pkg/values/validation/schemas.go

```diff
@@ -0,0 +1,306 @@
+//  kubernetes:
+//    title: kubernetes event bindings
+//    type: array
+//    additionalItems: false
+//    minItems: 1
+//    items:
+//      type: object
+//      additionalProperties: false
+//      required:
+//      - kind
+//      patternProperties:
+//        "^(watchEvent|executeHookOnEvent)$":
+//          type: array
+//          additionalItems: false
+//          minItems: 0
+//          items:
+//            type: string
+//            enum:
+//            - Added
+//            - Modified
+//            - Deleted
+//      properties:
+//        name:
+//          type: string
+//        apiVersion:
+//          type: string
+//        kind:
+//          type: string
+//        includeSnapshotsFrom:
+//          type: array
+//          additionalItems: false
+//          minItems: 1
+//          items:
+//            type: string
+//        queue:
+//          type: string
+//        jqFilter:
+//          type: string
+//          example: ".metadata.labels"
+//        keepFullObjectsInMemory:
+//          type: boolean
+//        allowFailure:
+//          type: boolean
+//        executeHookOnSynchronization:
+//          type: boolean
+//        waitForSynchronization:
+//          type: boolean
+//        resynchronizationPeriod:
+//          type: string
+//        nameSelector:
+//          "$ref": "#/definitions/nameSelector"
+//        labelSelector:
+//          "$ref": "#/definitions/labelSelector"
+//        fieldSelector:
+//          type: object
+//          additionalProperties: false
+//          required:
+//          - matchExpressions
+//          properties:
+//            matchExpressions:
+//              type: array
+//              items:
+//                type: object
+//                additionalProperties: false
+//                minProperties: 3
+//                maxProperties: 3
+//                properties:
+//                  field:
+//                    type: string
+//                  operator:
+//                    type: string
+//                    enum: ["=", "==", "Equals", "!=", "NotEquals"]
+//                  value:
+//                    type: string
+//        group:
+//          type: string
+//        namespace:
+//          type: object
+//          additionalProperties: false
+//          minProperties: 1
+//          maxProperties: 2
+//          properties:
+//            nameSelector:
+//              "$ref": "#/definitions/nameSelector"
+//            labelSelector:
+//              "$ref": "#/definitions/labelSelector"
+//`,
+//	"v0": `
+//type: object
+//additionalProperties: false
+//minProperties: 1
+//properties:
+//  onStartup:
+//    title: onStartup binding
+//    description: |
+//      the value is the order to sort onStartup hooks
+//    type: integer
+//  schedule:
+//    type: array
+//    items:
+//      type: object
+//  onKubernetesEvent:
+//    type: array
+//    items:
+//      type: object
+//`,
+//}
+
+type ValuesSchemaType string
```

Comment:
I can't find where these constants are used. Is it possible that they are redundant?

Reply:
These constants are used now.